src/renderer/bootstrap.tsx

```diff
@@ -1,17 +1,17 @@
 import "./components/app.scss";
-
+import "../common/libs-config";
```

Comment:
Just a note that import with side-effects (configure) might be a bad idea.

Reply:
Do you any have suggestions how to configure packages  instead?
Currently `immer`, for example, configured inside `storageHelper.ts` which is even more bad idea since it's global configuration.